assets/js/base/context/providers/cart-checkout/payment-events/index.tsx

```diff
@@ -61,16 +61,14 @@ export const PaymentEventsProvider = ( {
 			isCalculating: store.isCalculating(),
 		};
 	} );
-	const { isPaymentSuccess, isPaymentFinished, isPaymentProcessing } =
-		useSelect( ( select ) => {
-			const store = select( PAYMENT_STORE_KEY );
+	const { isPaymentReady } = useSelect( ( select ) => {
+		const store = select( PAYMENT_STORE_KEY );
 
-			return {
-				isPaymentSuccess: store.isPaymentSuccess(),
-				isPaymentFinished: store.isPaymentFinished(),
-				isPaymentProcessing: store.isPaymentProcessing(),
-			};
-		} );
+		return {
+			isPaymentReady: store.isPaymentReady(),
+			isPaymentProcessing: store.isPaymentProcessing(),
```

Comment:
This might be a good place to document inline what ready and processing represent. I would also swap them around so its more obvious processing comes before ready.

Reply:
Done